Assets/Scripts/Health/HealthBar.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+/// <summary>
+/// Exclusively manages an agent's health bar including its color
+/// This script is not involved with an object's actual health value
+/// </summary>
+public class HealthBar : MonoBehaviour
```

Comment:
Just wondering, does it need to be MonoBehaviour?

Reply:
For simplicity purposes and making resources available through the inspector, I'd say yes :)